src/Build/NuGet.Build.Packaging.Tasks/CreatePackage.cs

```diff
@@ -286,6 +289,16 @@ static VersionRange AggregateVersions(VersionRange aggregate, VersionRange next)
 			return versionSpec.ToVersionRange();
 		}
 
+		static List<string> AggregateIncludes(List<string> aggregate, string next)
```

Comment:
Maybe rename to `AggregateAssetsFlow`? Since it now does Includes and Excludes?

Reply:
Will do